src/components/common/ColonyHome/ColonyHomeLayout.tsx

```diff
@@ -48,30 +45,34 @@ const ColonyHomeLayout = ({
   showActions = true,
   onDomainChange = () => null,
 }: Props) => {
+  const { colony } = useColonyContext();
+
+  if (!colony) {
+    return null;
+  }
```

Comment:
Agree with you that this pattern is a bit annoying. 

I like the cleanliness of using the context hook everywhere, it cuts down on imports a lot.

Now if there was a way to not require this check and we would be golden _(but no, there's isn't really a good way around it -- other  than force casting types, or returning default values, none of which are "good enough")_

Reply:
Yes, I was facing exactly the same issue in my previous job :)